workers/unity/Packages/io.improbable.gdk.core/Systems/SpatialOSReceiveSystem.cs

```diff
@@ -42,6 +45,10 @@ protected override void OnUpdate()
                 Profiler.BeginSample("ApplyDiff Entity");
                 entitySystem.ApplyDiff(diff);
                 Profiler.EndSample();
+
+                Profiler.BeginSample("ApplyDiff Networking Statistics");
+                networkStatisticsSystem.ApplyDiff(diff);
```

Comment:
since the system is not enabled outside the editor - what happens when this method is called?

Reply:
This call is not compiled in due to the `Conditional` attribute - https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.conditionalattribute

Reply:
neato